clientv3/maintenance.go

```diff
@@ -50,6 +51,11 @@ type Maintenance interface {
 	// Status gets the status of the endpoint.
 	Status(ctx context.Context, endpoint string) (*StatusResponse, error)
 
+	// HashKV returns the state of backend database at the time of the RPC.
```

Comment:
s/the state of/a hash of the/
s/backend database/KV state/? it's not the entire backend...